### PR TITLE
Fix a broken link in struct AuthCodeSpotify

### PR DESCRIPTION
**Pull request by escwxyz**
## Description

This commit fixes the link to point to the correct  Spotify developer docs.

## Motivation and Context

The documentation for AuthCodeSpotify struct contained a broken link.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

Please also list any relevant details for your test configuration

- [ ] Test A: xxx
- [ ] Test B: xxx

## Is this change properly documented?

Please make sure you've properly documented the changes you're making.

Don't forget to add an entry to the CHANGELOG if necessary (new features, breaking changes, relevant internal improvements).

### CODE REVIEW

**Comment by ramsayleung** [comment]
The CI jobs failed, you should rebase the latest commit from the master branch, I have fixed this build errors.

File: src/auth_code.rs [hunk 1]
```diff
@@ -13,7 +13,7 @@ use std::sync::Arc;
 use maybe_async::maybe_async;
 use url::Url;
 
-/// The [Authorization Code Flow](reference) client for the Spotify API.
```
Comment (ramsayleung) [inline src/auth_code.rs:16]: Is it the correct link?

File: src/auth_code.rs [hunk 2]
```diff
@@ -13,7 +13,7 @@ use std::sync::Arc;
...
+/// The [Authorization Code Flow][reference] client for the Spotify API.
```
Comment (ramsayleung) [inline src/auth_code.rs:16]: Is it the correct link?

Comment (escwxyz) [reply src/auth_code.rs:16]: > Is it the correct link?

Hi, you can check out here https://docs.rs/rspotify/latest/rspotify/struct.AuthCodeSpotify.html, it points to a wrong url.

Comment (ramsayleung) [reply src/auth_code.rs:16]: Yes, it points to a wrong URL. But I think your updated one `[Authorization Code Flow][reference]` is also incorrect. The correct URL should be: https://developer.spotify.com/documentation/web-api/tutorials/code-flow

Comment (escwxyz) [reply src/auth_code.rs:16]: > Yes, it points to a wrong URL. But I think your updated one `[Authorization Code Flow][reference]` is also incorrect. The correct URL should be: [developer.spotify.com/documentation/web-api/tutorials/code-flow](https://developer.spotify.com/documentation/web-api/tutorials/code-flow)

Yes, you're right, the old link would be redirected to the link you provide though. Shall I make an update?

Comment (ramsayleung) [reply src/auth_code.rs:16]: Yes, feel free to do that :)

**Review by ramsayleung — APPROVED** [review]

**Pull request merged**